include/xsimd/config/xsimd_config.hpp

```diff
@@ -213,6 +213,9 @@
     #else
         #define XSIMD_WITH_NEON64 0
     #endif
+#elif defined(_MSC_VER) && defined(_M_ARM64)
```

Comment:
There's a specific section for MSVC workarounds at the bottom. Also, from [MSVC docs](https://docs.microsoft.com/en-us/cpp/intrinsics/arm-intrinsics?view=msvc-160#top) it is implied that NEON support is always available. Would it be possible to drop the 64-bit check as well?